### ISSUE TITLE
Game: DragonQuestBuilders Error

### ISSUE THREAD

**Issue by MichalSy** [yuzu-emu/yuzu#1227]
Version: Canary  | HEAD-c216901
Game crashes with the following message:
```
[  18.658817] HW.GPU <Error> video_core\command_processor.cpp:WriteReg:46: Special buffer methods other than Bind are not implemented
[  18.663150] Service.FS <Error> core\hle\service\filesystem\fsp_srv.cpp:OpenDataStorageByDataId:633: could not open data storage with title_id=0100000000000806, storage_id=03
```

**Comment by Hexagon12** [comment]
This title ID is the NG Bad Word list, according to SwitchBrew.

You need to dump this system archive from your Switch.

**Comment by MichalSy** [comment]
Could you Plesse leave more information about the Bad word list?

**Comment by DarkLordZach** [comment]
This was fixed by a combination of #1403, #1223, #1187 and finally #1095. All that's left is for you to dump your nand.

Requesting to close.
